test/testlib/minikube_utilities.go

```diff
@@ -244,6 +244,26 @@ func InjectTestVersion(t *testing.T, options *helm.Options) {
 	options.SetValues["nuodb.image.tag"] = image.Nuodb.Image.Tag
 }
 
+func OverrideUpgradeRepository(t *testing.T, options *helm.Options) {
+	dat, err := ioutil.ReadFile(UPGRADE_INJECT_FILE)
```

Comment:
Do we have an internal repository that has the same naming scheme as DockerHub? I know the internal registry that we use has some naming scheme like `registry.internal.nuodb.com/release/rel41-centos7:latest`.

Reply:
Looks like this is not an issue.